app/routines/update_user_contact_info.rb

```diff
@@ -101,6 +101,13 @@ def call
                            :other_school_type
                          end
 
+      if school != user.school.salesforce_id
```

Comment:
This may blow up if `user.school` is nil. If it doesn't blow up, it will always trigger because school is an ActiveRecord object and salesforce_id is a string but see below first